src/0x.ts

```diff
@@ -163,6 +163,12 @@ export class ZeroEx {
      * @return  An instance of the 0x.js ZeroEx class.
      */
     constructor(provider: Web3Provider, config?: ZeroExConfig) {
+        if (_.isUndefined((provider as any).sendAsync)) {
+            // Web3@1.0 provider doesn't support synchronous http requests,
+            // so it only has `send` method, instead of `send and `sendAsync` in web3@0.x.x
```

Comment:
`so it only has an async `send` method, instead of a `send and `sendAsync` in web3@0.x.x`